packages/cloud/src/queries/service-logs.ts

```diff
@@ -1,14 +1,15 @@
 import type { CreateServiceLog, ServiceLogType } from '@logto/schemas';
 import type { PostgreSql } from '@withtyped/postgres';
 import { sql } from '@withtyped/postgres';
-import type { Queryable } from '@withtyped/server';
+import type { JsonObject, Queryable } from '@withtyped/server';
 
 import { insertInto } from '#src/utils/query.js';
 
 export type ServiceLogsQueries = ReturnType<typeof createServiceLogsQueries>;
 
 export const createServiceLogsQueries = (client: Queryable<PostgreSql>) => {
-  const insertLog = async (data: Omit<CreateServiceLog, 'payload'>) =>
+  // TODO: @sijie make with-typed JsonObject compatible with unknown or unify JsonObject and ArbitraryObject
```

Comment:
`JsonObject` with strict type definition is by design. an arbitrary object will cause issues for queries

Reply:
Seems that replacing the `ArbitraryObject` in the DB schema definition with `JsonObject` can address this issue.

Reply:
Leave this change to upcoming PR to make the current change simple.